### ISSUE TITLE
App not responding with 3.0 and macOS Catalina

### ISSUE THREAD

**Issue by quentinlesceller** [milgra/macmediakeyforwarder#92]
Not sure if there is any logs that I can use but the app stopped responding when I updated it to 3.0 on macOS Catalina.

2.8 worked fine.

Let me know if I can anything to help debug this.

**Comment by tomkidd** [comment]
Yeah I just ran into this too and I can't help but wonder if the issue is that iTunes doesn't exist anymore :)

**Comment by egla89** [comment]
> Not sure if there is any logs that I can use but the app stopped responding when I updated it to 3.0 on macOS Catalina.
> 
> 2.8 worked fine.
> 
> Let me know if I can anything to help debug this.

Does it stop working, or does it not start? You need to launch it with CTRL+Click at first launch to allow it to run

**Comment by quentinlesceller** [comment]
> Does it stop working, or does it not start? You need to launch it with CTRL+Click at first launch to allow it to run

As mentioned above, for some reasons 2.8 worked. Upgraded with brew to 3.0 and suddenly app stop responding when launched.I'll post more debug Xcode log here.

**Comment by quentinlesceller** [comment]
I don't see any logs that could help. I assume it's a commit after https://github.com/milgra/macmediakeyforwarder/tree/c110aa97312874980dde84d79ff8e67f1feca94f. 
I've reverted to 2.8 (built locally) and it works fine.

**Comment by netsap** [comment]
I'm facing the same issue.

**Comment by netsap** [comment]
I managed to fix the issue by deleting and re adding the app in the accessibility view, adding the app to launch at login and restarting my laptop.

**Comment by quentinlesceller** [comment]
Same here thanks @netsap ! Closing.

**Comment by petr-fiala** [comment]
Issue still exists in 3.1 on Catalina, app not responding... Reinstalling app/removing from security>accessibility doesn't help.
Downloading older 2.8 solves the problem.
Issue shouldn't be closed.

**Comment by krunk4ever** [comment]
Couldn't get it working either, tried with both the latest version 3.1 as well as 2.8. Kept getting the `Not responding` error even after granting accessibility privilege for the app. Ended trying an alternative [beardedspice](https://github.com/beardedspice/beardedspice) which worked for me.
